Ginger/GingerCore/Drivers/WindowsLib/WindowsDriver.cs

```diff
@@ -105,7 +105,16 @@ public override void RunAction(Act act)
 
             if (actClass != "ActSwitchWindow" || (actClass == "ActWindow" && ((ActWindow)act).WindowActionType != ActWindow.eWindowActionType.Switch))
             {
-                CheckRetrySwitchWindowIsNeeded();
+                var checkWindow = true;
```

Comment:
why this flag? is it set to false for UI Element?

i think the retry is needed for a reason